nixos/modules/programs/fish.nix

```diff
@@ -157,11 +138,6 @@ in
       status --is-interactive; and not set -q __fish_nixos_interactive_config_sourced
       and begin
         ${fishAliases}
-
-        set --prepend fish_function_path ${pkgs.fishPlugins.foreign-env}/share/fish/vendor_functions.d
-        fenv source /etc/fish/foreign-env/interactiveShellInit > /dev/null
-        set -e fish_function_path[1]
```

Comment:
Same as in [nix-darwin](https://github.com/LnL7/nix-darwin/pull/270/files#r554594991), we need to keep these sourcings but should translate all those init files with babelfish

Reply:
Ah, I see, I was thinking `cfge` was the configuration for the module, but it's the environment configuration. I'll make the change to translate these with babelfish here and in nix-darwin.